wasm/src/graphics/context.rs

```diff
@@ -1,28 +1,32 @@
 use super::render::Render;
```

Comment:
The name Render is kinda misleading

Reply:
I think, we could replace the current `Context`-structure with the `Render`-structure, but call it `Render`. The current `Context` does only wrap the `Render`er, this abstraction should not be needed

Reply:
It's just that I fond myself confused multiple times when working on the codebase, because I would expect a struct to be called renderer an a function to be called render